browser/resources/sidebar/bookmarks/bookmark_folder.html

```diff
@@ -0,0 +1,239 @@
+    padding-block-end: 0;
+    padding-block-start: 0;
+    padding-inline-start: var(--row-padding-inline-start);
+    text-align: start;
+    width: 100%;
+  }
+
+  .row:not([empty]):hover,
+  .row:focus-visible:focus {
+    background-color: var(--cr-hover-background-color);
+    outline: none;
+  }
+
+  .row:not([empty]):active {
+    background-color: var(--cr-active-background-color);
+  }
+
+  .arrow,
+  .icon,
+  .title {
+    /**
+     * Keep content-visibility on inner children of .row instead of on .row
+     * as the CSS property seems to have bugs with rendering elements outside
+     * of .row when dragging elements around.
+     */
+    content-visibility: auto;
+  }
+
+  #arrow {
+    grid-area: arrow;
+    justify-self: center;
+  }
+
+  #arrowIcon {
+    --cr-icon-button-hover-background-color: transparent;
+    --cr-icon-button-active-background-color: transparent;
+    margin: 0;
+    transform: rotate(-90deg);
+    transition: transform 100ms cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  #arrowIcon[open] {
+    transform: rotate(0);
+  }
+
+  #arrowIcon:focus,
+  #arrowIcon:hover {
+    background: none;
+    box-shadow: none;
+  }
+
+  :host-context([dir='rtl']) #arrowIcon:not([open]) {
+    transform: rotate(90deg);
+  }
+
+  .icon {
+    background-position: left center;
+    background-repeat: no-repeat;
+    grid-area: icon;
+    height: 16px;
+    justify-self: center;
+    width: 16px;
+  }
+
+  #folderIcon {
+<if expr="is_win">
+    background-image: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_CLOSED_WIN_LIGHT);
+</if>
+<if expr="is_macosx">
+    background-image: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_CLOSED_LIGHT);
+</if>
+<if expr="is_linux">
+    background-image: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_CLOSED_LIN_LIGHT);
+</if>
+  }
+
+<if expr="not is_macosx">
+  #folderIcon {
+    height: 20px;
+    width: 20px;
+  }
+
+  #folderIcon[open] {
+<if expr="is_win">
+    background-image: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_OPEN_WIN_LIGHT);
+</if>
+<if expr="is_linux">
+    background-image: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_OPEN_LIN_LIGHT);
+</if>
+  }
+</if>
+
+  /**
+   * Folder icons are OS-specific. On Mac, there is a separate folder icon
+   * for dark mode; on non-Mac, there is a separate folder icon for open states
+   * and the icon for that is slightly larger. This is mirroring the icons
+   * used in chrome://bookmarks.
+   */
+  @media (prefers-color-scheme: dark) {
+    #folderIcon {
+<if expr="is_win">
+    background-image: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_CLOSED_WIN_DARK);
+</if>
+<if expr="is_macosx">
+      background-image: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_CLOSED_DARK);
+</if>
+<if expr="is_linux">
+    background-image: url(chrome://theme/IDR_BRAVE_BOOKMARK_FOLDER_CLOSED_LIN_DARK);
+</if>
+    }
+
+<if expr="not is_macosx">
+    #folderIcon {
+      height: 20px;
+      width: 20px;
+    }
+
+    #folderIcon[open] {
```

Comment:
Does macOS not have a folder open icon?

Reply:
Not sure why macOS has different with others.
Only macOS doesn't use folder open icon at all places. maybe it's macOS theme style?
bookmark manager on macOS also doesn't use open icon.